rero/rero-ils-ui
# locations: adapt detail view with new fields

* Adapts location resource detail view to display new fields defined into the json schema.
* Adapts "request item" select box to display only valid pickups location if needed
* Refactors 'generateTooltipMessage' function to be modular (working for other message type than 'deleteMessage')

Works with : https://github.com/rero/rero-ils/pull/708

## How to test?

- Edit a location to check edition of new fields "contact_email", "allow_request", "send_notification" and "restrict_pickup_to". Set some restrictions on "pickup_location" (choose one or two pickup locations). 
- Add an item for this location.
- Check the "request an item for a patron" for this item. The pikup location select box should only display the pickups you previously selected.

## Code review check list
